src/components/Calculator.css

```diff
@@ -0,0 +1,44 @@
+.calculator {
+  background-color: #edebeb;
+  width: 100%;
```

Comment:
- [ ] This is a desktop app, so please 🙏🏽 correct the width to be similar to the original design 👇🏽 

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/236062868-45a909ee-544b-4c1f-b4b9-957dfc37b397.png) | ![image](https://user-images.githubusercontent.com/58642949/236063018-5c3d0200-64fa-42f9-898a-9d21566b9f9d.png) |